docs/index.md

```diff
@@ -2,7 +2,7 @@
 
 Welcome to TestKube - your somewhat opinionated and friendly Kubernetes testing framework!
 
-TestKube decouples test artefacts and execution from CI/CD tooling; tests are meant to be part of your
+TestKube decouples test artifacts and execution from CI/CD tooling; tests are meant to be part of your
 clusters state and can be executed as needed:
```

Comment:
@exu , maybe there should not be a new line? I mean merge next with the present. Seems like the lines break is not intentional.

Reply:
👍🏻